evalml/data_checks/target_leakage_data_check.py

```diff
@@ -96,26 +96,30 @@ def validate(self, X, y):
             ...                   "code": "TARGET_LEAKAGE",
             ...                   "details": {"columns": ["leak"], "rows": None}}],
             ...     "errors": [],
-            ...     "actions": [{"code": "DROP_COL",
+            ...     "actions": {"action_list": [{"code": "DROP_COL",
             ...                  "data_check_name": "TargetLeakageDataCheck",
-            ...                  "metadata": {"columns": ["leak"], "rows": None}}]}
+            ...                  "metadata": {"columns": ["leak"], "rows": None}}], "default_action": None}}
 
             The default method can be changed to pearson from mutual information.
 
             >>> X['x'] = y / 2
             >>> target_leakage_check = TargetLeakageDataCheck(pct_corr_threshold=0.8, method='pearson')
             >>> assert target_leakage_check.validate(X, y) == {
-            ...     'warnings': [{'message': "Columns 'leak', 'x' are 80.0% or more correlated with the target",
+            ...     "warnings": [{'message': "Columns 'leak', 'x' are 80.0% or more correlated with the target",
```

Comment:
I love that you fixed this, but now the other quotes are inconsistent :sob:

Reply:
You're right, I must have done a replace-all somewhere--I'll go back and try to fix the other places 😭